export.js

```diff
@@ -76,6 +89,19 @@ _.extend(Avatar, {
     return (Avatar.options.cssClassPrefix)? Avatar.options.cssClassPrefix: 'avatar';
   },
 
+  getBackgroundColor: function (user) {
```

Comment:
I'm not sure how useful a default function with hard coded colours is here. I think it would be better to just put that as an example function in the documentation.